planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/heuristics/BlindSearch.java

```diff
@@ -50,7 +50,7 @@ public BlindSearch(int maxIter) {
       // value to or create a new method that does not consider the current one.
       // This way may help for replanning, when even the first attempt for
       // solution will be based on the current deployment
-      Solution[] bestSols = findSolutions(null, cloudOffers,numPlansToGenerate);
+      Solution[] bestSols = super.findInitialRandomSolutions(cloudOffers,numPlansToGenerate,topology);
```

Comment:
Done. Thanks. 
As you said, it was not necessary. I had kept them only to avoid confusion while I was doing the refactoring of these methods.